ansible/group_vars/all

```diff
@@ -34,10 +34,6 @@ runtimesManifest:
   defaultImageTag: "latest"
   runtimes:
     nodejs:
-    - kind: "nodejs"
```

Comment:
Hum I'm going to leave manifest alone, and maybe try skipping the docker pull if kind is deprecated